package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "3.0.5",
+  "version": "3.1.0",
```

Comment:
I updated the minor version number since this PR implements more than a patch but less than a major change.